common/test/acceptance/pages/studio/html_component_editor.py

```diff
@@ -7,25 +7,40 @@ class HtmlComponentEditorView(ComponentEditorView):
     """
     Represents the rendered view of an HTML component editor.
     """
-    def set_content_and_save(self, content):
+    def set_content_and_save(self, content, raw=False):
         """
         Types content into the html component and presses Save.
         """
-        self.set_content(content)
+        self.set_content(content, raw)
         self.save()
 
-    def set_content_and_cancel(self, content):
+    def set_content_and_cancel(self, content, raw=False):
         """
         Types content into the html component and presses Cancel to abort the change.
         """
-        self.set_content(content)
+        self.set_content(content, raw)
         self.cancel()
 
-    def set_content(self, content):
+    def set_content(self, content, raw=False):
         """
         Types content into the html component, leaving the component open.
+        Optionally edits in 'raw HTML' mode as indicated by raw param.
```

Comment:
Yay consistency! I went ahead and formatted all the method level docstrings in this file the same way.